e2e/tests/functional/plugins/plot/overlayPlot.e2e.spec.js

```diff
@@ -52,14 +52,9 @@ test.describe('Overlay Plot', () => {
         await page.locator('li.c-tree__item.menus-to-left .c-disclosure-triangle').click();
         await page.locator('.c-click-swatch--menu').click();
         await page.locator('.c-palette__item[style="background: rgb(255, 166, 61);"]').click();
-
         // gets color for swatch located in legend
-        const element = await page.waitForSelector('.plot-series-color-swatch');
-        const color = await element.evaluate((el) => {
-            return window.getComputedStyle(el).getPropertyValue('background-color');
-        });
-
-        expect(color).toBe('rgb(255, 166, 61)');
+        const seriesColorSwatch = page.locator('.gl-plot-label > .plot-series-color-swatch');
+        await expect(seriesColorSwatch).toHaveCSS('background-color', 'rgb(255, 166, 61)');
```

Comment:
this is awesome

Reply:
Yup!
<img width="632" alt="image" src="https://user-images.githubusercontent.com/9259993/233476500-c8966f8e-a86b-4a26-9675-e841a8ee7b4d.png">